scripts/status-rapids-repositories.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# Reports on the status of each RAPIDS repository.
+#
+# By default the short form of the status is reported, along with the branch
+# name (`-s -b`) - an alternative form can be reported by providing the
+# arguments to `git status` as arguments to this script.
+
+set -Eeo pipefail
+
+if [ ! -z $1 ]
+then
+    # Use arguments for status from user
+    ARGS="$@"
+else
+    # Default arguments: short-form changes with branch
+    ARGS="-s -b"
+fi
+
+cd $(dirname "$(realpath "$0")")/../../
```

Comment:
Does this need to be escaped? (I'm not 100% sure.)
```suggestion
cd $(dirname "$(realpath \"$0\")")/../../
```

Reply:
I'm not sure, and I have to confess my bash expertise is quite limited. The heritage of this script is that it's derived from [`pull-rapids-repositories.sh`](https://github.com/trxcllnt/rapids-compose/blob/master/scripts/pull-rapids-repositories.sh) which I didn't author, and doesn't escape those quotes, for whatever that's worth :slightly_smiling_face:

Reply:
It's probably fine to leave this as-is. If there are problems, we can fix it later.

Reply:
If you wanted to test an edge case, it would probably involve paths with spaces.